Source/Blazorise/Themes/Options/ThemeFontOptions.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Blazorise;
+
+/// <summary>
+/// Defines the font theme options.
+/// </summary>
+public record ThemeFontOptions
```

Comment:
Do we want to have some kind of smart types to help the user set these? or are we fine with "free edit"?

Reply:
What do you mean by smart types?

Reply:
Like the ones we have for other stuff but still allow strings.

Font.Family.Helvetiva, etc....
Font.Size.1px rem, etc...?
so on...

Reply:
Ah, that. No. It would be impossible to have every type. The user should be able to define any custom font family or size.

Reply:
Like I said, it should still allow introduction of custom strings. I think we have that somewhere? Icons?
You're right that it's ALOT of types, but we could have the most common.
Just an idea, of coursse it's way more work to make it work.